rust/tests/delta_arrow_test.rs

```diff
@@ -0,0 +1,14 @@
+extern crate deltalake;
+use arrow::datatypes::DataType as ArrowDataType;
+
+#[tokio::test]
+async fn test_arrow_from_delta_decimal_type() {
```

Comment:
Looks like there is no async function involved, so better define it as a sync test. It will get us a much more readable stack trace.

```suggestion
#[test]
fn test_arrow_from_delta_decimal_type() {
```

Reply:
Done!